frontend/src/lib/components/NotFound/index.tsx

```diff
@@ -17,6 +18,7 @@ interface NotFoundProps {
 export function NotFound({ object, caption }: NotFoundProps): JSX.Element {
     const { preflight } = useValues(preflightLogic)
     const { openSupportForm } = useActions(supportLogic)
+    const is3000 = useFeatureFlag('POSTHOG_3000')
```

Comment:
This is a multivariate flag now. I've adjusted the hook so that it can also handle those.
```suggestion
    const is3000 = useFeatureFlag('POSTHOG_3000', 'test')
```